README.md

```diff
@@ -161,13 +161,65 @@ Instead of using `mount()` from `enzyme` ...use this: `import {mountWithTheme} f
 We have decided to only use ECMAScript proposals that are in stage 3 (or later) (See [TC39 Proposals](https://github.com/tc39/proposals)). Additionally, because we are migrating to typescript, we will align with what their compiler supports.
 The only exception to this are decorators.
 
+## Optional Chaining and Null Coalescing
```

Comment:
Hmm maybe this header should be a bit more generic? Can we also add a link to the Quick Links suggestion at the top?

```suggestion
## New Syntax
```

Reply:
Done.